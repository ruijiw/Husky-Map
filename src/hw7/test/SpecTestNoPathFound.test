# Find a path using the LOLDictionary dataSet.
# This test uses the parser, the MarvelPaths2's setUpGraph
# and Dijkstra algorithm and applies to a targeted dataSet.
# This test is for one character does not have the path to
# another character.

LoadGraph dictionary LOLDictionary.tsv

FindPath dictionary RIVEN MORGANA

FindPath dictionary MORGANA KATARINA