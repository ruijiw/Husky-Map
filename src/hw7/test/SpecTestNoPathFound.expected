# Find a path using the LOLDictionary dataSet.
# This test uses the parser, the MarvelPaths2's setUpGraph
# and Dijkstra algorithm and applies to a targeted dataSet.
# This test is for one character does not have the path to
# another character.

loaded graph dictionary

path from RIVEN to MORGANA:
no path found

path from MORGANA to KATARINA:
no path found